recipe/build.sh

```diff
@@ -1,5 +1,5 @@
 mkdir build
 cd build
 export ISISROOT=$PWD
-cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -Disis3Data=/usgs/cpkgs/isis3/data -Disis3TestData=/usgs/cpkgs/isis3/testData ../isis
+cmake -GNinja -DJP2KFLAG=ON -Dpybindings=OFF -DKAKADU_INCLUDE_DIR=/isisData/kakadu -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$PREFIX -Disis3Data=/usgs/cpkgs/isis3/data -Disis3TestData=/usgs/cpkgs/isis3/testData ../isis
```

Comment:
It doesn't really matter for the conda recipe, since it doesn't run the tests, but it might be a good idea to update this to use the new `isis_data` and `isis_testData` data areas and replace `isis3Data` with `isisData` and `isis3TestData` with `isisTestData` so that this isn't confusing.

Reply:
Since they are not really needed I just removed the data vars and replaced them with buildTests=NO